Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -861,6 +861,15 @@ theorem IsCompact.exists_bound_of_continuousOn' [TopologicalSpace α] {s : Set 
 #align is_compact.exists_bound_of_continuous_on' IsCompact.exists_bound_of_continuousOn'
 #align is_compact.exists_bound_of_continuous_on IsCompact.exists_bound_of_continuousOn
 
+@[to_additive]
+theorem HasCompactMulSupport.exists_bound_of_continuous [TopologicalSpace α]
+    {f : α → E} (hf : HasCompactMulSupport f) (h'f : Continuous f) : ∃ C, ∀ x, ‖f x‖ ≤ C := by
+  rcases hf.exists_bound_of_continuousOn' h'f.continuousOn with ⟨C, hC⟩
+  refine ⟨max C 0, fun x ↦ ?_⟩
+  by_cases hx : x ∈ mulTSupport f
+  · exact (hC x hx).trans (le_max_left C 0)
+  · simp [image_eq_one_of_nmem_mulTSupport hx]
```

Comment:
I think it would be shorter to use [HasCompactMulSupport.isCompact_range](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Support.html#HasCompactMulSupport.isCompact_range), [IsCompact.bounded](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/MetricSpace/Basic.html#IsCompact.bounded), [Metric.Bounded.exists_norm_le](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/Normed/Group/Basic.html#Metric.Bounded.exists_norm_le) and [Set.forall_range_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Image.html#Set.forall_range_iff), but I haven't tried so I could be wrong.

Reply:
Indeed! My first version was along these lines, but I missed `Metric.Bounded.exists_norm_le'`. With it, this is a two-liner. Thanks!